test/e2e/apimachinery/resource_quota.go

```diff
@@ -135,6 +135,79 @@ var _ = SIGDescribe("ResourceQuota", func() {
 		framework.ExpectNoError(err)
 	})
 
+	// TODO: convert the test to conformance test once the loadbalancer node port control feature is GAed
+	ginkgo.It("should create a ResourceQuota and capture the life of a load balancer type service with allocateLoadBalancerNodePorts.", func() {
```

Comment:
This can be done with an integration test since we are not exercising the data path, it will be also much cleaner since we can make it trable driven.

Reply:
^ agree, if we can put this test into https://github.com/kubernetes/kubernetes/blob/master/test/integration/service/loadbalancer_test.go it would be ideal. But there's a question about whether this is a Service test or a ResourceQuota test

Reply:
Yeah I would say the intention was to test the node port quota but it requires to provision a LB type service. I can try add it in the quota integration test, it seems there's no service related tests for quota yet.

Reply:
Moved the e2e test to quota integration test, now it's passing.